src/main/java/acme/entities/lectures/Lecture.java

```diff
@@ -0,0 +1,46 @@
+
+package acme.entities.lectures;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.Length;
+import org.hibernate.validator.constraints.URL;
+
```

Comment:
añade los imports necesarios
```suggestion
import lombok.Getter;
import lombok.Setter;
import javax.persistence.Entity;
```